app/controllers/login.js

```diff
@@ -1,5 +1,7 @@
-import Ember from 'ember';
+import { Controller, computed } from 'ember';
 
 import OsfTokenLoginControllerMixin from 'ember-osf/mixins/osf-token-login-controller';
 
-export default Ember.Controller.extend(OsfTokenLoginControllerMixin);
+export default Controller.extend(OsfTokenLoginControllerMixin, {
+    login: computed.alias('model'),
```

Comment:
My personal preference would be to nix the rule altogether in the COS config. Rather than just bikeshed, here is the thinking:

1. The `model` hook has a clear origin; aliasing it adds a layer of obfuscation for where data comes from- especially for how we currently write pages.
2. Per linter claim re routable components, I can't find solid support for that statement in [the spec](https://github.com/ef4/rfcs/blob/routeable-components/active/0000-routeable-components.md). Also, the spec has been the subject of [much back-and-forth](https://emberjs.com/blog/2017/04/05/emberconf-2017-state-of-the-union.html) and currently lacks a clear champion. We'd be adding complexity to support an API that will almost certainly change before release.

`https://github.com/emberjs/rfcs/pull/38`